src/components/Request/index.examples.js

```diff
@@ -7,16 +7,25 @@ import Request from './index'
 const Comments = ({comments}) => (
   <div>
     {map(comments, (comment, index) => (
-      <div key={index}>
-        <div>By {comment.email}</div>
-        <div>{comment.body}</div>
-        <hr />
+      <div
+        key={index}
+        className='pb3'
+      >
+        <div className='b'>
+          By {comment.email}
+        </div>
+        <div className='gray'>
+          {comment.body}
+        </div>
       </div>
     ))}
   </div>
 )
 
+const decoratorClasses = 'mh-vh-100 bg-navy white pa3'
+
 storiesOf('Request')
+  .addDecorator((story) => (<div className={decoratorClasses}>{story()}</div>))
```

Comment:
In the future, would be good to replace all these `addDecorator` styles in examples with a global `StyleReset` component that can be used to reset the `body` in consuming apps as well. Here is an issue for it:
https://github.com/eggheadio/egghead-instructor-center/issues/119